subsys/testsuite/ztest/src/ztest_new.c

```diff
@@ -62,6 +62,13 @@ static void end_report(void)
 	}
 }
 
+#ifdef CONFIG_ZTEST_SHELL
```

Comment:
hmm, ztest_new.c should not be there, must have forgotten to remove it when old ztest was removed, so not sure about all those changes and why they were needed, strange.

Reply:
I was just trying to be consistent. But yea was kind of surprised to see it still here.